core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -299,50 +247,65 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                 layout-wrap
                 class="metrics"
                 layout-align="space-between center">
+              <a ng-class="checkMobileView() ? 'oppia-dashboard-mobile-statistics-card-link': 'oppia-dashboard-statistics-card-link'"
```

Comment:
Is it possible to unify the private/public cases further? It seems to me that both have an outer link, and I'm not sure why that outer link has to be replicated twice.